app/src/sharedTest/java/org/oppia/app/profile/AdminAuthActivityTest.kt

```diff
@@ -132,6 +135,115 @@ class AdminAuthActivityTest {
     }
   }
 
+  @Test
+  fun testAdminAuthActivity_inputTextPreservedOnRotation() {
+    launch<AdminAuthActivity>(
+      AdminAuthActivity.createAdminAuthActivityIntent(
+        context,
+        "12345",
+        -10710042
+      )
+    ).use {
+      onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_pin)))).perform(
+        typeText("12345"),
+        closeSoftKeyboard()
+      )
+      it.onActivity {
+        it.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+        assert(it.requestedOrientation == ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
+      }
+      onView(withId(R.id.submit_button)).check { view, noViewFoundException ->
+        view.background = context.getDrawable(R.drawable.state_button_primary_background)
+      }
+      it.onActivity {
+        it.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
+        assert(it.requestedOrientation == ActivityInfo.SCREEN_ORIENTATION_PORTRAIT)
+      }
+      onView(withId(R.id.submit_button)).check { view, noViewFoundException ->
+        view.background = context.getDrawable(R.drawable.state_button_primary_background)
+      }
+    }
+  }
+
+  @Test
+  fun testAdminAuthActivity_buttonStateIsPreservedOnRotation() {
+    launch<AdminAuthActivity>(
+      AdminAuthActivity.createAdminAuthActivityIntent(
+        context,
+        "12345",
+        -10710042
+      )
+    ).use {
+      onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_pin)))).perform(
+        typeText("12345"),
+        closeSoftKeyboard()
+      )
+      it.onActivity {
+        it.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+        assert(it.requestedOrientation == ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
+      }
+      onView(allOf(withId(R.id.input), isDescendantOfA(withId(R.id.input_pin)))).check(
+        matches(
+          withText("12345")
+        )
+      )
+    }
+  }
+
+  @Test
+  fun testAdminAuthActivity_inputCorrectPasswordLandscape_opensAddProfileActivity() {
```

Comment:
We can remove this test case. It is not needed.

Reply:
Done